apps/cookbook/src/app/showcase/segmented-control-showcase/segmented-control-showcase.component.html

```diff
@@ -165,6 +165,21 @@ <h3>Keyboard support</h3>
     <kbd>End</kbd> moves focus to the last segment button
   </li>
 </ul>
+<h3>
+  Provide custom
+  <code>aria-label</code>
+  for segment buttons
```

Comment:
Consider something that signals the 'Why'/ overall idea rather than specific implementation details. 
```suggestion
  Accessible names for segment buttons
```